.hooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+printf "\n✅ pre-commit hook\n\n"
```

Comment:
what's the point of this? didn't it used to do a `composer lint` or something similar? I know I've had to run `composer fix` to commit before.

Reply:
To have it in place - the linting etc has been moved to `pre-push`.